src/librustdoc/clean/utils.rs

```diff
@@ -124,11 +124,7 @@ pub(crate) fn ty_args_to_args<'tcx>(
             )))
         }
         GenericArgKind::Const(ct) => {
-            // FIXME(effects): this relies on the host effect being called `host`, which users could also name
-            // their const generics.
-            // FIXME(effects): this causes `host = true` and `host = false` generics to also be emitted.
-            if let ty::ConstKind::Param(p) = ct.kind()
-                && p.name == sym::host
+            if let ty::GenericParamDefKind::Const { is_host_effect: true, .. } = params[index].kind
```

Comment:
I *could* add a test for “`impl Trait<true>`”. However, it was actually fixed by #112463 (sic!). The `/*host*/ false` case is unreachable atm, #117530 would change that).

Re. testing `sym::host` vs `#[rustc_host]`, I don't think it's worth adding a `#![feature(rustc_attrs)]` test in `tests/rustdoc/`. Doesn't feel right.

Reply:
I'd prefer to have a test, even if pointless. If everything is covered, regressions will be easier to track down.